zephyr/west.yml

```diff
@@ -0,0 +1,39 @@
+# The west manifest file for downstream Zephyr module ZXA.
```

Comment:
Please move this to the root, as `west.yml`. We really really do not need this to be in a separate folder, let's use the default whenever we can to simplify things and avoid additional command-line options.

Reply:
See rationale above for why it *shouldn't* be in the top level in this example.